lib/diagnostic.js

```diff
@@ -29,8 +33,13 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
+const any = function (workingArray, callback) {
 
+    if(callback(element, index, array)) {
```

Comment:
The logic here is correct, but you need to iterate through the array.  As it's currently written, you are trying to call callback with three arguments that don't exist.